melanoma_phd/database/PatientDatabase.py

```diff
@@ -316,6 +316,10 @@ def __create_variables_from_config(
                             **list(iteration_variable_config.values())[0],
                         )
                         created_variables[reference_variable.id] = reference_variable
+                except KeyError as error:
+                    raise ValueError(
+                        f"Database loading error: the next config variables could not been loaded causing the database loading abortation [{errors}]"
```

Comment:
The concept here is if there are some variable loading failing, they go to `ValueError` exception and saved to `errors` list. But later on, if we catch a `KeyError` most probably is because the previous variables that caused the `ValueError` were not been insterted to `created_variables` dictionary, so here we are printing the saved errors in `errors` list. However, I'll also add to the print the concrete catched `error` 👍